src/components/table/SnapshotAccessRequestTable.tsx

```diff
@@ -58,7 +59,13 @@ function SnapshotAccessRequestTable({
     {
       label: 'Request Name',
       name: 'snapshotName',
-      render: (row: SnapshotAccessRequestResponse) => row.snapshotName || '',
+      render: (row: SnapshotAccessRequestResponse) => (
+        <ModalTrigger
```

Comment:
I agree that this isn't `Modal` but `ModalTrigger` also doesn't seem right to me. This is rendering a prompt (which looks like a hyperlink) that renders its content into a modal dialog. Maybe `ModalLink` would be better?